go/analyzer/staticcheck/util/util.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+
+	"golang.org/x/tools/go/analysis"
+	"honnef.co/go/tools/analysis/lint"
+	"honnef.co/go/tools/quickfix"
+	"honnef.co/go/tools/simple"
+	"honnef.co/go/tools/staticcheck"
+	"honnef.co/go/tools/stylecheck"
+)
+
+var StaticcheckAnalyzers = [][]*lint.Analyzer{
```

Comment:
these don't need to be public, and I think I'm general it's a good idea to keep the package scope slim. Could we move this into the function?

Reply:
I think it's a bit easier to read to have it in a separate package scoped var and encourage others to modify.

But I will just move them into the function as you suggested for now, no strong opinion here.